app/views/admin/kudos/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Editing Admin Kudo</h1>
+
+<%= render 'form', admin_kudo: @kudo %>
```

Comment:
Inside __form_ you use `@kudo`, not `admin_kudo`, so get rid of this assignement, or use local variable in _form view - same thing applies to _new_ view